.env

```
@@ -0,0 +1,21 @@
+# AFFINE_SERVER_HOST=affine.yourdomain.com
+# or 
+# AFFINE_SERVER_EXTERNAL_URL=https://affine.yourdomain.com
+
+# Folder locations
+DB_DATA_LOCATION=./postgres
+UPLOAD_LOCATION=./storage
+CONFIG_LOCATION=./config
+
+# Database credentials
+DB_USERNAME=affine
+DB_PASSWORD=affine123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USERNAME key

(UnorderedKey)